java/src/jmri/jmrit/beantable/BlockTableAction.java

```diff
@@ -982,14 +987,16 @@ void okPressed(ActionEvent e) {
             }
         }
         String user = userName.getText();
+        String uName = user; // result stored separately to prevent incremental manipulation
```

Comment:
I'm not sure what this "result stored separately to prevent incremental manipulation" comment means. What's "incremental manipulation"

Would it make more sense to have both copies be the normalized form?  The goal is to present that back to the user, so that eventually people catch on.

Reply:
Instead of 'incremental', I should have said 'recursive'; code comment updated.
This is not what you would want to get adding a range of 5 Blocks and entering 1 as the starting address:
![recursive_block_naming](https://user-images.githubusercontent.com/14811327/39961008-a67861c6-562d-11e8-90ad-c381063c2fca.png)